app/experimenter/templates/base.html

```diff
@@ -62,6 +54,18 @@ <h3 class="pt-2">
               Leave Feedback
             </a>
           </div>
+          <div class="col text-right">
+            <div>
+              <span class="fas fa-user"></span>
+              {{ request.user }}
+            </div>
+
+            <div>
+              <a class="nocolorstyle" href="{% url "home" %}?owner={{ request.user.id }}&amp;archived=on">
+                {{ request.user.owned_experiments.count }} Owned Experiment{{ request.user.experiment_set.count|pluralize:"s" }}
```

Comment:
You're counting two different things here. One is `owned_experiments.count` and one is `experiment_set.count`. I believe that's why the zero isn't getting pluralized as it should in the screenshot.

Reply:
Also, `|pluralize` and `|pluralize:"s"` are equivalent since the `"s"` is the default. It could be dropped if you prefer.